tox.ini

```diff
@@ -11,6 +11,7 @@ deps = -r{toxinidir}/requirements.txt
 basepython = python3
 whitelist_externals = bash
 commands =
+  python3 ./tests/scripts/missing_keys.py
```

Comment:
lets create a new entry point in tox, maybe
```
tox -enodepool
```
then we can move this script into tools folder (need to create). This follows kinda of what we do in project-config